server/dal/src/storage_engine.rs

```diff
@@ -231,32 +209,15 @@ impl Storage {
                                         }}
                                     ]
                                 }}"#,
-                                bucket_name
-                            ))
-                            .send()
-                            .await
-                            .expect("failed to set bucket policy");
-
-                        // Stop the timer
-                        let duration = start.elapsed();
-
-                        // Log the duration of the bucket creation
-                        info!(
```

Comment:
Please log in some way that a new bucket was created

Reply:
Buckets are no longer created :) It's on AWS at least impossible without throwing permissions wide open

Reply:
You can still log that the permissions have been updated or something. Anyhow, is there a way to initialize MinIO with a bucket on first run? If we want to keep this project attractive to develop for it should run out of the box in one way or another

Reply:
Good point.

I'll add an option prod/dev or something like that. To distinguish minio vs aws.

Reply:
You can also add a env variable like S3_CREATE_BUCKET_ON_STARTUP=false that is used with MinIO (just like S3_FORCE_PATH_STYLE=true)